jiant/models.py

```diff
@@ -687,6 +692,30 @@ def build_single_sentence_module(task, d_inp: int, project_before_pooling: bool,
     return module
 
 
+def build_sop(task, d_inp, model, params, args):
+    """
+    Build and load the pretrained head for the sentence order prediction task.
+    Right now, there is only support for ALBERT.
```

Comment:
We should add an assert or warning for this. Do we expect reasonable (non-standard) behavior with other pretrained models?

Reply:
We do load the weights from the pooling layer which only makes sense for ALBERT, but there shouldn't be anything here that breaks for any other model. I'm hesitant to say we support other models before we run experiments with other models, so I'll just add an assert.